xla/service/gpu/gpu_fused_mha_runner.cc

```diff
@@ -201,20 +201,21 @@ void AssignSeed(GpufMHAConfig &config,
 }
 
 template <typename ElementType, typename OutputType>
-Status RunFusedMHABackward(GpufMHABackwardParams params, se::Stream *stream,
```

Comment:
Could we aim to minimize the diff, here and in other places?

Reply:
fixed